Examples/Tests/nuclear_fusion/analysis_proton_boron_fusion.py

```diff
@@ -316,20 +316,20 @@ def astrophysical_factor(E):
 def pb_cross_section_buck_fit(E):
     ## E is in MeV
     ## Returns cross section in b using a power law fit of the data presented in Buck et al.,
-    ## Nuclear Physics A, 398(2), 189-202 (1983) in the range E > 3.5 MeV.
-    E_start_fit = 3.5
+    ## Nuclear Physics A, 398(2), 189-202 (1983) in the range E > 9.76 MeV.
+    E_start_fit = 9.76
     ## Cross section at E = E_start_fit = 3.5 MeV
-    cross_section_start_fit = 0.2168440845211521
+    cross_section_start_fit = 0.01277998
     slope_fit = -2.661840717596765
     return cross_section_start_fit*(E/E_start_fit)**slope_fit
 
 def pb_cross_section(E):
     ## E is in keV
-    ## Returns cross section in b using the fits described in W.M. Nevins and R. Swain,
-    ## Nuclear Fusion, 40, 865 (2000) for E < 3.5 MeV and a power law fit of the data presented in
-    ## Buck et al., Nuclear Physics A, 398(2), 189-202 (1983) for E > 3.5 MeV.
+    ## Returns cross section in b using the fits described in A. Tentori and F. Belloni,
+    ## Nucl. Fusion, 63, 086001 (2023) for E < 9.76 MeV otherwise returns a power law fit
+    ## decribed in Buck et al., Nuclear Physics A, 398(2), 189-202 (1983)
```

Comment:
```suggestion
    ## of the data in Buck et al., Nuclear Physics A, 398(2), 189-202 (1983)
```

The fit itself is not described in the paper, it's just a fit of the data I had done when implementing the pB fusion module.